geomstats/_backend/numpy/__init__.py

```diff
@@ -12,6 +12,7 @@
     arccos,
     arccosh,
     arcsin,
+    arctan,
```

Comment:
a similar line needs to be added to the pytorch and tensorflow init scripts

Reply:
Done.